<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowForm</class>
 <widget class="QMainWindow" name="MainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,1">
    <item>
     <widget class="QLineEdit" name="smiles">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
        <item>
         <widget class="QScrollArea" name="scrollArea2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QSvgWidget" name="svg2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>82</width>
             <height>143</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSim">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QSvgWidget" name="svg">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>227</width>
               <height>129</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tbl">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bFirst">
            <property name="text">
             <string>First</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lbl">
            <property name="maxLength">
             <number>15</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bNext">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bLast">
            <property name="text">
             <string>Last</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="console">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximumBlockCount">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportSDF"/>
   <addaction name="actionImportSmiles"/>
   <addaction name="actSim"/>
   <addaction name="actionDeleteAll"/>
   <addaction name="separator"/>
   <addaction name="actionSmiles"/>
   <addaction name="actionConsole"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionImportSDF">
   <property name="text">
    <string>importSDF</string>
   </property>
   <property name="toolTip">
    <string>Import SDF</string>
   </property>
  </action>
  <action name="actionImportSmiles">
   <property name="text">
    <string>importSmiles</string>
   </property>
   <property name="toolTip">
    <string>Import SMILES</string>
   </property>
  </action>
  <action name="actionSmiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Smiles</string>
   </property>
   <property name="toolTip">
    <string>SMILES</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>actionAbout</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="text">
    <string>deleteAll</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Console</string>
   </property>
  </action>
  <action name="actSim">
   <property name="text">
    <string>Sim</string>
   </property>
   <property name="toolTip">
    <string>Similarity</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtSvg</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
